xla/tools/compute_cost.cc

```diff
@@ -43,18 +44,22 @@ The input file can be obtained from XProf graph viewer by clicking
 
 Usage:
 
-  bazel run compute_cost -- --input=path/to/hlo_module --format=[hlo|pb|pbtxt] [--gpu]
+  bazel run compute_cost -- --input=path/to/hlo_module --format=[hlo|pb|pbtxt] [--gpu] [--all]
 )";
 }  // namespace
 
 int main(int argc, char** argv) {
   std::string input, format;
   bool gpu = false;
+  bool all = false;
   std::vector<tsl::Flag> flag_list = {
       tsl::Flag("input", &input, "input file"),
       tsl::Flag("format", &format, "hlo|pb|pbtxt"),
       tsl::Flag("gpu", &gpu,
-                "Use GPU flavor of cost analysis instead of the generic one")};
+                "Use GPU flavor of cost analysis instead of the generic one"),
+      tsl::Flag("all", &all,
+                "Print costs and deduplicated name of each instruction, not "
```

Comment:
This suggests that you print two things:
- the cost of each instruciton
- the total costs.

However, the code below prints either one or the other. Please make the description and code consistent. I would prefer a change to the code where you print the individual costs for each instruction and then also print the totals at the end.

Reply:
done